Makefile

```diff
@@ -16,6 +16,10 @@ BUILD_IMAGE_PREFIX ?= ""
 # other than DockerHub
 RUN_IMAGE_PREFIX ?= ""
 
+# Prefix for all images used for PSM realated tests, enabling use with
+# registries other than DockerHub
```

Comment:
realated -> related. DockerHub -> Docker Hub. I have already made the second change for other occurrences.